lib/opentracing/nil_tracer.rb

```diff
@@ -0,0 +1,16 @@
+module OpenTracing
+  class NilTracer
+    # Start a new span
+    # @param operation_name [String] The name of the operation represented by the span
+    # @param child_of [Span] A span to be used as the ChildOf reference
+    # @param start_time [Time] the start time of the span
+    # @param tags [Hash] Starting tags for the span
+    def start_span(operation_name, child_of: nil, start_time: nil, tags: nil)
```

Comment:
doesn't this need to return a noop Span instance? or maybe ruby is one of those languages that doesn't have NPEs? :)